### PR TITLE
fixFewTests

### PR DESCRIPTION
**Pull request by xinlian12**
Fix few failed tests in ci pipeline

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM - Thanks!

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/ClientTelemetryTest.java [hunk 1]
```diff
@@ -248,7 +247,7 @@ public void httpClientTests(CosmosClient cosmosClient) throws Exception {
         assertThat(clientTelemetryHttpClient).isNotSameAs(clientTelemetryMetadataHttpClient);
 
         // Test metadataHttpClient is used for IMDS requests
-        HttpClientUnderTestWrapper clientTelemetryMetadataHttpClientWrapper = new HttpClientUnderTestWrapper(clientTelemetryHttpClient);
+        HttpClientUnderTestWrapper clientTelemetryMetadataHttpClientWrapper = new HttpClientUnderTestWrapper(clientTelemetryMetadataHttpClient);
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/ClientTelemetryTest.java:250]: Thanks for fixing these @xinlian12  :)

**Review by kushagraThapar — APPROVED** [review]
Changes look good to me, thanks @xinlian12 !!

**Pull request merged**